libs/ui-toolkit/src/components/notification/notification.tsx

```diff
@@ -41,11 +48,20 @@ export const Notification = ({
         {
           'border-gray-700 dark:border-gray-300': intent === Intent.None,
           'border-vega-blue': intent === Intent.Primary,
-          'border-vega-green dark:border-vega-green': intent === Intent.Success,
-          'border-yellow-500': intent === Intent.Warning,
+          'border-vega-green-550 dark:border-vega-green':
+            intent === Intent.Success,
+          'border-vega-orange': intent === Intent.Warning,
           'border-vega-pink': intent === Intent.Danger,
         },
-        'border rounded text-xs p-4 flex items-start gap-2.5 bg-neutral-100 dark:bg-neutral-900',
+        {
+          'bg-vega-light-100 dark:bg-vega-dark-100 ': intent === Intent.None,
+          'bg-vega-blue-300 dark:bg-vega-blue-700': intent === Intent.Primary,
+          'bg-vega-green-300 dark:bg-vega-green-700': intent === Intent.Success,
+          'bg-vega-orange-300 dark:bg-vega-orange-650':
+            intent === Intent.Warning,
+          'bg-vega-pink-300 dark:bg-vega-pink-650': intent === Intent.Danger,
+        },
+        'border rounded p-2 flex items-start gap-2.5 bg-neutral-100 dark:bg-neutral-900 my-4',
         className
```

Comment:
here we have two different styles for background when the only last one matters. Is this intentional?

Reply:
updated